package.json

```diff
@@ -23,7 +23,7 @@
     "test:unit:jest": "jest",
     "test:unit:watch": "npm run test:unit -- --watch",
     "test:unit:nerdctl-stub": "cd ./src/go/nerdctl-stub/ && go test ./...",
-    "test:e2e": "node ./scripts/e2e.mjs",
+    "test:e2e": "cross-env NODE_ENV=test node ./scripts/e2e.mjs",
```

Comment:
That script is supposed to [set `NODE_ENV=test`](https://github.com/rancher-sandbox/rancher-desktop/blob/main/scripts/e2e.mjs#L91); if it's not doing so there's a bug somewhere.

Reply:
Oh I see @mook-as. It should do the trick properly.

I've tried to use `cross-env` in order to export the `NODE_ENV` across the OS distros. 
Adding the NODE_ENV=test, the failure below became more less intermittent, but haven't fixed for good:
![Screen Shot 2021-09-30 at 2 47 17 PM](https://user-images.githubusercontent.com/37411123/135527904-a370db18-3bb3-44a0-aaa0-ff27fb4bb213.png)


*jest logs*
```bash
> rancher-desktop@0.5.0 test:e2e
> node ./scripts/e2e.mjs

[error] (node:92876) [DEP0148] DeprecationWarning: Use of deprecated folder mapping "./" in the "exports" field module resolution of the package at /Users/evertonperes/projects/rancher-desktop/node_modules/@nuxt/components/package.json.
Update this package.json to use a subpath pattern like "./*".
(Use `node --trace-deprecation ...` to show where the warning was created)
Browserslist: caniuse-lite is outdated. Please run:
npx browserslist@latest --update-db

Why you should do it regularly:
https://github.com/browserslist/browserslist#browsers-data-updating
Hash: 18d60a5e6e50713ddca3
Version: webpack 4.46.0
Time: 15196ms
Built at: 2021-09-30 2:46:46 p.m.
            Asset      Size      Chunks                   Chunk Names
    background.js  1.14 MiB  background  [emitted]        background
background.js.map  1.33 MiB  background  [emitted] [dev]  background
Entrypoint background = background.js background.js.map
[./background.ts] 15.3 KiB {background} [built]
[./node_modules/electron-devtools-installer/dist/index.js] 6.24 KiB {background} [built]
[./src/config/settings.ts] 7.24 KiB {background} [built]
[./src/k8s-engine/k8s.ts] 863 bytes {background} [built]
[./src/main/kim.ts] 5.74 KiB {background} [built]
[./src/main/mainEvents.ts] 259 bytes {background} [built]
[./src/main/mainmenu.ts] 3.86 KiB {background} [built]
[./src/main/networking/index.ts] 3.48 KiB {background} [built]
[./src/main/paths.ts] 9.44 KiB {background} [built]
[./src/main/tray.ts] 8.7 KiB {background} [built]
[./src/main/update/index.ts] 4.13 KiB {background} [built]
[./src/resources.ts] 1.04 KiB {background} [built]
[./src/utils/childProcess.ts] 3.29 KiB {background} [built]
[./src/utils/latch.ts] 1.29 KiB {background} [built]
[./src/utils/logging.ts] 3.35 KiB {background} [built]
    + 264 hidden modules
ts-jest[versions] (WARN) Version 4.2.3 of typescript installed has not been tested with ts-jest. If you're experiencing issues, consider using a supported version (>=2.7.0 <4.0.0). Please do not report issues in ts-jest if you are using unsupported versions.
(node:92891) [DEP0148] DeprecationWarning: Use of deprecated folder mapping "./" in the "exports" field module resolution of the package at /Users/evertonperes/projects/rancher-desktop/node_modules/yaml/package.json.
Update this package.json to use a subpath pattern like "./*".
(Use `node --trace-deprecation ...` to show where the warning was created)
 FAIL  e2e/main.e2e.spec.ts (7.221s)
  Rancher Desktop
    ✕ opens the window (3ms)
    ✕ should display welcome message in general tab (1ms)
    ✕ should switch to Kubernetes Settings tab
    ✕ should switch to Port Forwarding tab (1ms)
    ✕ should switch to Images tab
    ✕ should switch to Troubleshooting tab

  ● Rancher Desktop › opens the window

    javascript error: javascript error: Cannot read property 'type' of null
      (Session info: chrome=89.0.4389.128)

      24 |
      25 |   beforeAll(async() => {
    > 26 |     await app.start();
         |     ^
      27 |     browserWindow = app.browserWindow;
      28 |     navBarPage = new NavBarPage(app);
      29 |   });

      at Object.getErrorFromResponseBody (node_modules/webdriver/build/utils.js:94:12)
      at WebDriverRequest._request (node_modules/webdriver/build/request.js:134:31)
      at Browser.wrapCommandFn (node_modules/@wdio/utils/build/shim.js:63:29)
      at Browser.wrapCommandFn (node_modules/@wdio/utils/build/shim.js:63:29)
      at Object.<anonymous> (e2e/main.e2e.spec.ts:26:5)
```
Not sure if we need to implement Node integration as spectron requires: https://github.com/electron-userland/spectron/blob/master/README.md#node-integration